packages/ui/src/components/createInitiativeForm.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @author Sandhya
+ * @name Login
+ * @desc This component renders the login page
+ */
+
+import React from 'react';
+import { Card } from 'material-ui/Card';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import DatePicker from 'material-ui/DatePicker';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+const CreateInitiativeForm = ({
+  onSubmit,
+  onChange,
+  errors,
+  user,
+  pastDateCheck,
+  currentDate= new Date(),
+  totalParticipants,
+  onParticipantDropDownChange
+
+}) => (
+  <Card className="container create-initiative-page">
+  <form onSubmit={onSubmit}>
+      <h2 className="card-heading">Create Initiative Form</h2>
+      <div className="field-line">
+        <TextField
+          floatingLabelText="Initiative Name"
+          name="initiativeName"
+          onChange={onChange}
+          value={user.initiativeName}
+          errorText={errors.initiativeName}
+        />
+      </div>
+      <div>
+        <DatePicker hintText="Initiative start date" 
+        shouldDisableDate={pastDateCheck(currentDate)}/>
+      </div>
+      <div>
+        <DatePicker hintText="Initiative end date" 
+        shouldDisableDate={pastDateCheck(currentDate)}
+        />
+      </div>
+      <div>
+        <SelectField
+            floatingLabelText="Participants"
+            value={totalParticipants}
+            onChange={(event, index, value) => onParticipantDropDownChange(event, index, value)}
+            autoWidth={true}
+          >
+            <MenuItem value={1} primaryText="Never" />
```

Comment:
On selecting an option from the dropdown, it doesn't show up in the UI area.
A better idea here would be to create a constant with various options and iterate through the constant and render the MenuItem, cause options are gonna be dynamic and not hard coded